.github/workflows/kind.yml

```diff
@@ -0,0 +1,26 @@
+name: Kind e2e Tests
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  test-e2e:
+    runs-on: ubuntu-latest
```

Comment:
@moshloop I switched back to the VM. 
Seem simpler/faster for now to use the VM - which is basically with [everything and the kitchen sink](https://github.com/actions/virtual-environments/blob/master/images/linux/Ubuntu1804-README.md) already installed - than to get Kind working in a docker setup *and* updating build tools to the latest git and pushing to docker.